### ISSUE TITLE
Cannot anymore remove ephemeral property of a message

### ISSUE THREAD

**Issue by Nyfos** [discordjs/discord.js#7968]
### Which package is this bug report for?

discord.js

### Issue description

Steps to reproduce:
1. Receive a command interaction
2. Defer the reply with an ephemeral message
3. Edit the reply by removing the ephemeral property
4. The message remains ephemeral

The strange thing here is that I can't find any "ephemeral" property in the documentation or in autocompletion... I haven't changed my code, it was working correctly before.

*I've set issue priority to "Medium" because it's pretty annoying, above all if everybody has this bug...*

### Code sample

```typescript
client.on("interactionCreate", async interaction => {
  await interaction.deferReply({ ephemeral: true })
  interaction.editReply({ content: "Bouh, ephemeral.", ephemeral: false })
})
```

### Package version

13.6.0

### Node.js version

16.14.0

### Operating system

Windows

### Priority this issue should have

Medium (should be fixed soon)

### Which partials do you have configured?

No Partials

### Which gateway intents are you subscribing to?

Guilds, GuildBans, GuildEmojisAndStickers, GuildIntegrations, GuildWebhooks, GuildInvites, GuildPresences, GuildMessages, GuildMessageReactions

**Comment by didinele** [comment]
Discord has never supported this to my knowledge, so I'm a bit doubtful that it "worked for you" at some point.

**Comment by imranbarbhuiya** [comment]
Yeah editReply options don't have any `ephemeral` property as discord doesn't allow changing ephemeral flag on `editReply`

**Comment by Nyfos** [comment]
That's strange.. I was sending an ephemeral deferring before responding to each command... If I can't anymore do that, how can I have ephemeral commands and non-ephemeral commands without checking if the command needs to be ephemeral? I am displaying sensitive informations, I need to be able to switch between ephemeral and public message...

**Comment by Nyfos** [comment]
Okay. Thank you so much for your help!
